core/auth.py

```
@@ -0,0 +1,567 @@
+
+
+class LoginRequest(BaseModel):
+    """Login request for JWT token."""
+    username: str
+    password: str
+
+
+class TokenResponse(BaseModel):
+    """JWT token response."""
+    access_token: str
+    token_type: str = "bearer"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "token_type"

(S105)